dbt/include/duckdb/macros/materializations/external.sql

```diff
@@ -76,9 +82,10 @@
   {{ drop_relation_if_exists(temp_relation) }}
 
   -- register table into glue
+  --I dont know glue so can you explain me a bit about that?
```

Comment:
There's a lot of history here-- the ability to register an external table with the AWS Glue catalog (and thus make it accessible to e.g. AWS Athena) actually predates the concept of dbt-duckdb plugins (in the `BasePlugin` sense.) The original idea of plugins for external relations was to generalize this capability to other cataloging systems.

Reply:
Nice, I see; I will need you to support me from the requirements point of view because I am unsure if I can imagine all possible use cases implemented until now. I don't have so much experience in the AWS world so more azure, google cloud but happy to learn about glue and athena
I will skip this one till we don't agree on some structure of the code; then, we can speak about details and each use case.